Examples/Python/setup.sh.in

```diff
@@ -10,17 +10,26 @@
 # robust way.
 
 if [ -n "$ZSH_VERSION" ]; then
-    export PYTHONPATH=${0:a:h}:$PYTHONPATH
+    python_dir=${0:a:h}
 elif [ -n "$BASH_VERSION" ]; then
     python_dir=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}"  )" &> /dev/null && pwd  )
-    export PYTHONPATH=$python_dir:$PYTHONPATH
 else
     # If the current shell is not ZSH or Bash, we can't guarantee that the
     # script will work, so we throw an error.
     echo "ERROR:   neither ZSH nor Bash was detected, other shells are not supported. The environment has not been modified."
     exit 1
 fi
 
+# Make python package available
+export PYTHONPATH=$python_dir:$PYTHONPATH
+
+# Make ODD available if possible
+odd_dir=$python_dir/../thirdparty/OpenDataDetector/factory
+if [ -d "$odd_dir" ]; then
```

Comment:
`"$OSTYPE" == "darwin"*` for bash I guess, not sure what to do on zsh.

Reply:
For me both `zsh` and `bash` give `linux-gnu` for `$OSTYPE`, so I would go with that I think

Reply:
What you could do is : `export LD_LIBRARY_PATH=$odd_dir:$LD_LIBRARY_PATH` in the case of `linux-gnu` and `export DYLD_LIBRARY_PATH+=:$odd_dir export DD4HEP_LIBRARY_PATH+=:$odd_dir` in the case of `darwin` ? That way it is useful to everybody ?